### ISSUE TITLE
Latex inline support?

### ISSUE THREAD

**Issue by jcguu95** [polymode/polymode#278]
I'm looking for auto mode switching for inline latex codes, for example

```
Normal mode normal mode $latex mode$ normal mode
$$ latex mode,$$
normal mode normal mode 
\[
latex mode
\]
```

Is it currently supported to do this?

**Comment by bryce-carson** [comment]
I don't believe it's included in the box, but once the semester finishes up I might look into it. I'm _**not**_ someone who typically hacks on Elisp, but I'd like in-line & set-off or stand-alone (out-line? 🤔) L<sup>A</sup>T<sub>E</sub>X.

Given that it's necessary for me to include L<sup>A</sup>T<sub>E</sub>X in my Rmd documents especially, I would love this feature.

With the _modus-themes_, or _Spacemacs_ themes, there may be a problem with the faces being applied. For me, however, there is improper handling of a string such as this in my Rmd buffer:
```latex
\Delta H = \frac{4N_{e}\mu}{1 + 4N_{e}\mu}
```
The string would be in-line, surrounded by $s. Strings like this appear as below.
![Screenshot from 2021-03-16 01-12-16](https://user-images.githubusercontent.com/51974245/111269918-d19a5900-85f4-11eb-97f7-365942fac682.png)

It seems like faces "spill over" as well. Compare the appearance of the Rmd code block below with the screenshot of the same code in an Emacs polymode buffer.
```Rmd
The effective population size ($N_{e}$) was either $1000$ or $10000$, and
in the first half of the simulation, $10N_{e}$ generations, a
```
![Screenshot from 2021-03-16 01-17-35](https://user-images.githubusercontent.com/51974245/111270407-6e5cf680-85f5-11eb-9032-e585122ee007.png)

A fuller support for Rmd would be *lovely*. I use csl files for my bibliography, so citation syntax' edge cases need to be handled as well.
- `[adapted from @Barrett2008]`
- `[ -@Savolainen2013 ]`

**Comment by vspinu** [comment]
@jcguu95 for which mode do you need inline latex support? There is already inline latex as part of the [markdown](https://github.com/polymode/poly-markdown/blob/1536cf0c32f71d5cd05c90f7905905e38006e95d/poly-markdown.el#L108-L135). 

@bryce-carson inline math in polymode is activated when you activate it in the markdown proper. See `markdown-enable-math`.

**Comment by bryce-carson** [comment]
> @bryce-carson inline math in polymode is activated when you activate it in the markdown proper. See `markdown-enable-math`. 

Thank you, I'll look at that variable.
